Bottom: 7ca527dff8040c8165692dedbb8bce3b168f3d7c
Top:    7ca527dff8040c8165692dedbb8bce3b168f3d7c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 04:40:09 +0100

Add configure checks for GtkGLEext, add to CFLAGS and LIBS

*** TODO: Need separate tests for libgl and libglut etc.. ***















---


